app/controllers/sequences_controller.rb

```diff
@@ -114,6 +114,29 @@ def show
 
     @title = "Tryptic peptide analysis of #{@original_sequence}"
 
+    @pept_with_go = 0
```

Comment:
I think this counting code can be made shorter (and clearer variable names?) or should be moved to a separate function. Maybe http://ruby-doc.org/core-2.5.0/Enumerable.html#method-i-group_by can help.

_[Original comment](https://github.ugent.be/unipept/unipept/pull/625#discussion_r13730) by @bmesuere on Mon Feb 19 2018 at 14:40._